### DISCUSSION TITLE
Have you seen GoAdmin? (Idea for being able to extend the Admin panel)

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#1062] [Q&A] [answered]
Goadmin design might be able to offer insight on how to offer extensibility in the admin panel.

https://github.com/GoAdminGroup/go-admin
https://demo.go-admin.com/admin/login

**Comment by ganigeorgiev** [comment, marked as answer]
I'll research the topic in more details sometime in the future (most likely together with the eventual workflow builder), but for now the [other tasks from the roadmap](https://github.com/orgs/pocketbase/projects/2) are with higher priority.

**Reply by dschissler** [reply]
For me, I saw GoAdmin a while ago but it didn't seem like a good idea to get stuck into using their user and admin system when it wasn't really offering me anything else other than a cool UI.  I think though that Pocketbase is unique though because it's offering (as of 0.8) the most essential and hardest part up front, and with enough barebones but well polished admin panel stuff to tie it all together.  So for me, since I already need this stuff then I'm interested in setting up more stuff the Pocketbase way since my most essential needs have already been met.

I think that svelte is also the magic puzzle piece here because it makes the size overhead so tiny for adding completely separate mini apps to an existing program.  It could be possible to add in custom Svelte entries which are admin panel tabs delivered as ES6 modules via dynamic import, and without needing to care about the framework bootstrap file sizes.  It's basically free.  For me Svelte's greatest offering is being able to tie together many disjoint mini apps in a way that isn't gross, and is actually quite elegant.

This would allow companies to use Pocketbase as their core and then add in their own product configuration panels to go with their custom routes.  It's then all shipped as a single binary with assets and everything.
